include/aspect/utilities.h

```diff
@@ -192,6 +192,21 @@ namespace aspect
                                                       double phi );   //longitude (radians)
 
     /**
+     * A struct to enable numerical output with a comma as thousands separator
+     */
+    struct ThousandSep : std::numpunct<char>
+    {
+      char do_thousands_sep() const
+      {
+        return ',';
+      }
+      std::string do_grouping() const
```

Comment:
same here.

In fact, both of these are also `protected` in the base class, see http://en.cppreference.com/w/cpp/locale/numpunct . There is nothing inherently wrong in making them `public` in your class, but it is not necessary, so I would also make them `protected` here.